### PR TITLE
DATA-416/create-a-derived-table

### PR DESCRIPTION
**Pull request by tamsinforbes**
Check links and lists format correctly, update and add to information.

### CODE REVIEW

File: source/documentation/tools/create-a-derived-table/index.md [hunk 1]
```diff
@@ -65,6 +62,8 @@ Create a Derived Table is a service that brings dbt, Git, and data access contro
 
 We’re still in beta so we'd love to get some of you using Create a Derived Table to get your feedback to help guide best practice data modelling in the Ministry of Justice and make sure we can continue to improve the user experience.
```
Comment (parminder-thindal-moj) [inline source/documentation/tools/create-a-derived-table/index.md:63]: I would probably rewrite to "help guide data modelling best practice,"

File: source/documentation/tools/create-a-derived-table/index.md [hunk 2]
```diff
@@ -265,7 +256,7 @@ If you're working on a data model as part of a team you can take inspiration fro
```
Comment (parminder-thindal-moj) [inline source/documentation/tools/create-a-derived-table/index.md:256]: On L255 (_it wont let me comment there for some reason_) spelling mistake "_changrs_"

**Review by parminder-thindal-moj — COMMENTED** [review]
[MISSING WORD] Couldn't add a comment on the exact line - but at L417, “team” is missing from let the data modelling know.

[SUGGESTION] The guidance doesn’t actually explain what “_marts_” are. Could always link to the [dbt docs](https://docs.getdbt.com/guides/best-practices/how-we-structure/4-marts), but users might not do that. I would probably just write 1-2 lines of what it is.

[SUGGESTION] I would add hyper links to the sidebar, for each of the different sections. It was abit frustrating scrolling backwards and forwards to check something in a section that I had read earlier.

I haven’t had a chance to test all of the instructions but most of them look good to me.

Also alot of it follows what Darius told me before he left. Nice job!

**Comment by parminder-thindal-moj** [comment]
Some error messages, you might want to incorporate in the docs somewhere, probably the FAQS, for why isn't my table appearing or something?

Screenshot 2023-03-08 at 11 28 37

Screenshot 2023-03-08 at 11 28 42

**Comment by tamsinforbes** [comment]
> Some error messages, you might want to incorporate in the docs somewhere, probably the FAQS, for why isn't my table appearing or something?
> 
> Screenshot 2023-03-08 at 11 28 37
> 
> Screenshot 2023-03-08 at 11 28 42

You are very welcome to add this to the troubleshooting page yourself, preferably with a proposed solution. It'll be in this page:

user-guidance/source/documentation/tools/create-a-derived-table/troubleshooting.md

**Comment by tamsinforbes** [comment]
> [SUGGESTION] The guidance doesn’t actually explain what “_marts_” are. Could always link to the [dbt docs](https://docs.getdbt.com/guides/best-practices/how-we-structure/4-marts), but users might not do that. I would probably just write 1-2 lines of what it is.
> 

the Data Modelling concepts sections is intending to cover a brief intro to data modelling concepts and jargon. Hopefully someone from the data modelling team will have time to work on this soon.

**Review by parminder-thindal-moj — APPROVED** [review]
Looks good to me. The only thing I haven't checked, is the Nav Side bar links, which requires a Ruby install. As it took you a while to set that up, i'll approve this in the meantime, just incase I encounter similar issues.

**Pull request merged**